src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -13,6 +13,44 @@ import * as Utils from 'src/libs/utils'
 import { Component } from 'src/libs/wrapped-components'
 
 
+class StatusLabel extends Component {
+  render() {
+    const { status } = this.props
+
+    return h(Fragment, [
+      status,
+      Utils.cond(
+        [status === 'Stopped', () => this.renderStartIcon()],
+        [status === 'Running', () => this.renderStopIcon()],
+        () => null
+      )
+    ])
+  }
+
+  renderStartIcon() {
+    const { clusterName, googleProject, refresh } = this.props
+
+    return h(Interactive,
+      {
+        as: icon('play'), class: 'is-solid', title: 'Start Cluster',
+        style: { color: Style.colors.primary, cursor: 'pointer' },
```

Comment:
I _believe_ you don't need to specify `cursor: 'pointer'` for an `Interactive` when there's an `onClick`.